packages/icons/src/library/inbox.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const inbox = (
+	<SVG xmlns="http://www.w3.org/2000/SVG" viewBox="0 0 24 24">
+		<path fill-rule="evenodd" d="M6 5.5h12a.5.5 0 01.5.5v7H14a2 2 0 11-4 0H5.5V6a.5.5 0 01.5-.5zm-.5 9V18a.5.5 0 00.5.5h12a.5.5 0 00.5-.5v-3.5h-3.337a3.5 3.5 0 01-6.326 0H5.5zM4 13V6a2 2 0 012-2h12a2 2 0 012 2v12a2 2 0 01-2 2H6a2 2 0 01-2-2v-5z" clip-rule="evenodd"/>
```

Comment:
Do we actually need the fill-rule here?

Reply:
Yes, without it you get this:

![fill](https://cldup.com/z1H46JlAGM-3000x3000.png)